### PR TITLE
[stable/cockroachdb] Allow custom annotations in StatefulSet

### LINKED ISSUE #13399: [stable/cockroachdb] Allow ExtraAnnotations on the CockroachDB Statefulsets
**Is this a request for help?**:

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

FEATURE REQUEST

Allow to add extra annotations on the CockroachDB Statefulsets for instance to control ServiceMesh injection etc (e.g. Linkerd2)

**Version of Helm and Kubernetes**:

2.13.1 / 1.14.1

**Which chart**:

stable/cockroachdb

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by christianhuening**
#### What this PR does / why we need it:

Adds ExtraAnnotation option to helm chart

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #13399

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/cockroachdb/values.yaml [hunk 1]
```diff
@@ -78,3 +78,8 @@ ExtraSecretMounts: []
 # - name: COCKROACH_ENGINE_MAX_SYNC_DURATION
 #   value: "24h"
 ExtraEnvArgs: []
+# ExtraAnnotations is an object to provide additional annotations to the Statefulset
+# e.g.:
+# ExtraAnnotations:
+#   key: values
+ExtraAnnotations: {}
```
Comment (a-robinson) [inline stable/cockroachdb/values.yaml]: You need to put the newline back at the end of the file to pass CI

**Comment by christianhuening** [comment]
@a-robinson This should be good now. I rebased this on the latest changes for 19.1.0 and bumbed the version again.

**Pull request merged**